apps/perf-test/src/scenarioRenderTypes.js

```diff
@@ -13,6 +13,8 @@ const DefaultRenderTypes = ['mount'];
 
 const scenarioRenderTypes = {
   ThemeProvider: AllRenderTypes,
+  GroupedList: DefaultRenderTypes,
+  GroupedListV2: DefaultRenderTypes,
```

Comment:
Do you need to add these here? The comment above seems to imply that if not specified the scenarios will run with `DefaultRenderTypes`.

Reply:
Looks like I don't need em.